index.js

```diff
@@ -0,0 +1,28 @@
+//const serverless = require('serverless-http');
+const express = require('express');
+const path = require('path');
+const app = express();
+//const router = express.Router();
+
+const PORT = 3000;
```

Comment:
Здесь можно порт указать таким образом:
```suggestion
const PORT = process.env.PORT || 3000; 
```
Тогда будет возможность брать значение из env-переменных, а уже если там её не окажется, то брать значение по умолчанию